src/hotspot/share/code/nmethod.hpp

```diff
@@ -135,8 +135,6 @@ class PcDescContainer : public CHeapObj<mtCode> {
 //
 // An nmethod contains:
 //  - header                 (the nmethod structure)
-//  [Relocation]
-//  - relocation information
```

Comment:
Whole this comment have to be re-written to reflect new layout and additional data sections outside CodeCache.